driver-async/src/main/com/mongodb/async/client/gridfs/helpers/AsyncStreamHelper.java

```diff
@@ -79,6 +79,12 @@ public void read(final ByteBuffer dstByteBuffer, final SingleResultCallback<Inte
                 transferDataFromByteBuffers(srcByteBuffer, dstByteBuffer, callback);
             }
 
+            @Override
+            public void skip(final long bytesToSkip, final SingleResultCallback<Long> callback) {
```

Comment:
Shouldn't this skip bytesToSkip in srcByteBuffer?

Reply:
As these are helpers / wrappers for reading information into GridFS, I don't see a usecase for skipping via the API. As such we don't break contract and report we skipped 0.